config/envs/default.py

```diff
@@ -11,6 +11,7 @@
 class DefaultConfig(object):
     FLASK_ROOT = str(Path(__file__).parent.parent.parent)
     FLASK_ENV = CommonConfig.FLASK_ENV
+    ENABLE_VALIDATION_LOGGING = True
```

Comment:
I think we want this turned off by default. We also need to be able to turn it on via an environment variable